cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4086,12 +4086,14 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = index=* sourcetype!="WinRegistry" NOT (sourcetype="WinEventLog" user_type="computer") (user=* OR User=* OR UserName=* OR Username=* OR userName=* OR username=* OR user_name=* OR src_user=*) \
+search = index=* sourcetype!="WinRegistry" NOT (sourcetype="WinEventLog" user_type="computer") NOT (sourcetype="meraki:accesspoints" (NOT eventData.identity="*")) (user=* OR User=* OR UserName=* OR Username=* OR userName=* OR username=* OR user_name=* OR src_user=*) \
 | `cs_user_inventory_data_filter` \
-| eval user=coalesce(src_user, user, user_name, username,Username, User, UserName, userName) \
-| search NOT user IN ("$*", "-", "' or '1=1", "") \
-| eval user_type=coalesce(user_type, user_role, user_category, "n/a") \
+| eval user=mvdedup(mvappend(coalesce(src_user, user, user_name, username,Username, User, UserName, userName), mail)), user_type=coalesce(user_type, user_role, user_category, "n/a") \
+| search NOT user IN ("$*", "-", "' or '1=1", ".{jndi", "") \
+| nomv user \
 | stats latest(user_type) as user_type values(index) as index values(sourcetype) as sourcetype latest(_indextime) as indextime by user vendor_product \
+| makemv delim="\
+" user \
```

Comment:
delim in multiline is it expected?

Reply:
Yes, **nomv** command add the newline between each values hence while converting it to multivalue using **makemv** delim should be new line. 
Here delim is supporting string not the regex hence i added new line as a string.